railties/lib/rails/generators/rails/app/templates/test/system_test_helper.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class ActionSystemTestCase < ActionSystemTest::Base
+  teardown do
```

Comment:
IMHO, adding default `driven_by :selenium, using: :chrome, on: 21800, screen_size: [ 1400, 1400 ]` here will be beneficial while still maintaining Rails conventions :), also this will act as self documentation to users who start to use `SystemTests` without need to hop thought documentation immediately

```ruby
require "test_helper"

class ActionSystemTestCase < ActionSystemTest::Base
  driven_by :selenium, using: :chrome, on: 21800, screen_size: [ 1400, 1400 ]

  teardown do
    take_failed_screenshot
    Capybara.reset_sessions!
  end
end
```